var/spack/repos/builtin/packages/rarpd/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Rarpd(MakefilePackage):
+    """RARP (Reverse Address Resolution Protocol) is a protocol which allows
+    individual devices on an IP network to get their own IP addresses from the
+    RARP server.  Some machines (e.g. SPARC boxes) use this protocol instead
+    of e.g. DHCP to query their IP addresses during network bootup."""
+
+    homepage = "https://github.com/OpenMandrivaAssociation/rarpd"
+
+    maintainers = ["jcpunk"]
+
+    version("981107", url="http://archive.ubuntu.com/ubuntu/pool/universe/r/rarpd/rarpd_0.981107.orig.tar.gz", sha256="e925a8c98501d3f92d11d15286538cb5228087b9ea0633e68a0c47d2d2242b56")
+
+    patch("001_from_0.981107_4.diff")
+    patch("002_from_0.981107_6.diff")
+    patch("003_from_0.981107_7.diff")
+    patch("010_standard_conventions.diff")
+    patch("015_notsock_and_debugging.diff")
+    patch("017_respond_to_arp.diff")
```

Comment:
1. (Minor request) Can you add a one-line comment above each patch to state what the patch is doing?
2. (Comment, optional) If these patches are available from a remote host we can use the full URL and avoid storing it in the built-in repo

Reply:
I've added in comments.  The patches aren't really seperated out in the remote locations.  Mostly the debian diff file has them included, but I'm not sure how to make spack extract the patches from that diff and make sense of them...

Reply:
Comments added for the patches.